addons/project/controllers/portal.py

```diff
@@ -31,15 +31,15 @@ def _prepare_portal_layout_values(self):
     def _project_get_page_view_values(self, project, access_token, **kwargs):
         values = {
             'page_name': 'project',
-            'project': project,
+            'project_id': project.id,
```

Comment:
For future development, it could be good to keep the object and not only the id. because if you need, later, in the xml template to add some values from the project, you will have to revert your changes anyway. In the template 'portal_my_project', you can use project.id instead of project_id